Resources/Prototypes/Entities/Objects/Misc/handcuffs.yml

```diff
@@ -52,6 +52,7 @@
       path: /Audio/Items/Handcuffs/rope_breakout.ogg
     startBreakoutSound:
       path: /Audio/Items/Handcuffs/rope_takeoff.ogg
+    uncuffMassMultiplies: true
```

Comment:
```suggestion
    uncuffMassMultiplies: true # Inverts the mass multiplier for handcuffs; larger has an easier time to break out while smaller struggles.
```

Reply:
Use a better field name, don't duplicate comments everywhere it's used

Reply:
real-pilled??????